src/oc_erchef/apps/chef_objects/src/chef_node.erl

```diff
@@ -159,7 +159,12 @@ ejson_for_indexing(#chef_node{name = Name, environment = Environment}, Node) ->
                                    {<<"chef_environment">>, Environment},
                                    {<<"recipe">>, extract_recipes(RunList)},
```

Comment:
Let's extract roles/recipes once and assign, then use it in both places instead of extracting twice.